docker/cluster.py

```diff
@@ -54,8 +54,20 @@ def create_cluster(network_name, number_of_nodes, public_key_path, dev, pull):
         docker_client.images.pull('couchbase/mobile-testkit', tag='latest')
 
     # Create docker network with name
-    log_info('Creating bridged network: {} ...'.format(network_name))
-    network = docker_client.networks.create(network_name)
+    log_info('Checking if network ({}) exists ...'.format(network_name))
+    conflicting_networks = [net for net in docker_client.networks.list() if net.name == 'cbl']
+
+    # If there is a network defined with the name of one you are trying to create
+    # Dev mode: reuse the existing network
+    # Strict mode: raise an exception
+    if len(conflicting_networks) > 0:
+        if not dev:
+            raise DockerError('ERROR!! Network already defined')
```

Comment:
Can we also provide an override to go ahead and delete the network and create a new one?